lib/speck.ex

```diff
@@ -224,4 +224,13 @@ defmodule Speck do
   defp apply_valid_values({value, error}, _valid_values) do
     {value, error}
   end
+
+  defp get_raw_value(map, key) do
+    raw_value = map[to_string(key)]
+
+    case raw_value do
+      nil -> map[key]
+      _ -> raw_value
```

Comment:
Alignment

Reply:
Fixed in cb146eb23d9ef9181a3f6c302a6c44cdf348b03e